src/ngraph/runtime/cpu/cpu_executor.cpp

```diff
@@ -68,13 +68,11 @@ namespace ngraph
                     for (int i = 0; i < num_thread_pools; i++)
                     {
                         int num_threads_per_pool;
-#if defined(EIGEN_OPENMP)
+
                         // Eigen threadpool will still be used for reductions
                         // and other tensor operations that dont use a parallelFor
-                        num_threads_per_pool = 1;
-#else
```

Comment:
Any idea why it was set to 1 ?

Reply:
It was based on an incorrect understanding of EIGEN_OPENMP on my part. When this was set I assumed all Eigen execution is going to go through openmp and hence reduced the number of threads in the eigen thread pool.